doc/howto/cluster_recover.md

```diff
@@ -100,12 +105,24 @@ Log on to each cluster member and complete the following steps:
 
    You can edit the addresses and the roles.
 
-After doing the changes on all cluster members, start the LXD daemon on all members again.
-For example, if you're using the snap:
+1. When the cluster configuration has been changed on one member, LXD will create
+   a tarball of the global database (`/var/snap/lxd/common/lxd/database/lxd_recovery_db.tar.gz`
+   for snap installations or `/var/lib/lxd/database/lxd_recovery_db.tar.gz`).
+   Copy this recovery tarball to the same path on all remaining cluster members.
+
+   ```{note}
+   The tarball can be removed from the first member after it is generated, but
```

Comment:
Wont it be reapplied on next start up?

Reply:
Yes; I wasn't sure if I wanted to include this note, since it makes no difference whether the "template member" loads the tarball it generated or not. I figured it was better to be explicit about it.

Reply:
Ty for the convo this morning. To clarify: the tarball is removed after it has been used to replace the global database, so it will not be reapplied on subsequent startups.

The CLI checks to make sure that the server isn't running before reconfiguring the db: [1](https://github.com/canonical/lxd/blob/main/lxd/main_cluster.go#L123) [2](https://github.com/canonical/lxd/blob/main/lxd/main_cluster.go#L389).

Reply:
Thanks, is this ready for review again?

Reply:
Sorry, ran out of time on Friday and had to run. Not sure if we discussed this or not but MicroCluster takes an automatic backup of the database dir before resetting the dqlite Raft log; I am going to drop a note in the recovery doc warning the user to take a backup of the database dir before performing the cluster recovery. If you'd prefer, I can implement a similar automated backup.

Reply:
Yeah i think that would be a good idea. Thanks!